README.md

```diff
@@ -62,7 +62,7 @@ facts. There are just so many facts. You can just say `dog fact` (or
 | ----------------- | ------------------------------------------------- |
 | `alan fact`       | Shares a fact about people named Alan             |
 | `alumni dog fact` | Shares a fact about a dog belonging to a TTS alum |
-| `ask ally`        | Shares a fact about accessible web development    |
+| `ask ally fact`   | Shares a fact about accessible web development    |
```

Comment:
[nonblocking] should it be `ask ally fact/ask a11y fact` or something to indicate you can use `a11y`?

Reply:
Ooh - thanks for catching this. It's a good question and I don't have the answer. I don't know if the original intention behind the regex is that a user can use either or to catch users who might accidentally use `a11y` or `ally` not remembering which one it was. 

I'll think about the best way to document the behavior and follow up in another pr!